### PR TITLE
CO-2414 Add mail source to model request

### PR DESCRIPTION
**Pull request by eicher31**

### CODE REVIEW

File: crm_request/models/request.py [hunk 1]
```diff
@@ -49,7 +52,12 @@ def message_new(self, msg, custom_values=None):
         """
         if custom_values is None:
             custom_values = {}
-        custom_values.update({'description': msg.get('body')})
+        txt = msg.get('to')
+        alias_char = txt[txt.find('"') + 1:txt.find("@")]
```
Comment (ecino) [inline crm_request/models/request.py]: That could possibly fail if the `to` field contains another format or unexpected destination. In that case all operation will be aborted which could lead to losing information.
```suggestion
        alias_char = txt.split("@")[0].replace('"', '')
```

File: crm_request/views/request.xml [hunk 2]
```diff
@@ -18,6 +18,10 @@
                 <attribute name="string">Description</attribute>
             </xpath>
 
+            <field name="team_id" position="after">
+                <field name="alias_id" groups="base.group_no_one"/>
```
Comment (ecino) [inline crm_request/views/request.xml:94]: Why adding group in a specific field? I would just let the field security be the same as the whole view no?

**Review by ecino — CHANGES_REQUESTED** [review]

**Comment by ecino** [comment]
Conflicts with changes from Christopher

File: crm_request/views/request.xml (hunk 2 above)
Comment (eicher31) [reply crm_request/views/request.xml:94]: I find that is a technical feature that pollute the view for the user.

File: crm_request/models/request.py [hunk 3]
```diff
@@ -55,9 +59,15 @@ def message_new(self, msg, custom_values=None):
         """
         if custom_values is None:
             custom_values = {}
+
+        alias_char = parseaddr(msg.get('to'))[1].split('@')[0]
```
Comment (eicher31) [inline crm_request/models/request.py:76]: @ecino Even better... ;-)

**Comment by ecino** [comment]
Conflicts to resolve.

**Comment by ecino** [comment]
Please resolve this...

**Review by ecino — APPROVED** [review]

**Pull request merged**